src/com/hrabrov/introduction_to_java/Main.java

```diff
@@ -0,0 +1,40 @@
+package com.hrabrov.introduction_to_java;
+
+import com.hrabrov.introduction_to_java.algorithmization.one_dimensional_array.OneDimensionalArray;
+import com.hrabrov.introduction_to_java.basic_of_software_code_development.branching.BranchingRunner;
+import com.hrabrov.introduction_to_java.basic_of_software_code_development.cycle.CycleRunner;
+import com.hrabrov.introduction_to_java.basic_of_software_code_development.linear_program.LinerRunner;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("*****    Start basic of software code development module     *****");
+        basicOFSoftwareCodeDevelopmentRunner();
+        System.out.println("*****    End basic of software code development module    *****\n");
+
+        System.out.println("*****    Start Algorithmization module     *****");
+        algorithmizationRunner();
+        System.out.println("*****    End Algorithmization module    *****");
+    }
+
+    public static void basicOFSoftwareCodeDevelopmentRunner() {
```

Comment:
You don't need it to be public because you only use those methods in the internal method. Please see and follow [SOLID principals](https://en.wikipedia.org/wiki/SOLID)